src/SprykerEco/Zed/Payone/Persistence/PayoneQueryContainer.php

```diff
@@ -44,7 +48,7 @@ public function createCurrentSequenceNumberQuery(int $transactionId): SpyPayment
      *
      * @return \Orm\Zed\Payone\Persistence\SpyPaymentPayoneQuery
      */
-    public function createPaymentByTransactionIdQuery($transactionId)
+    public function createPaymentByTransactionIdQuery($transactionId): SpyPaymentPayoneQuery
```

Comment:
It's a part of public API, please remove added type hints.

Reply:
Fixed.